consent/strategy_logout_test.go

```diff
@@ -490,4 +507,20 @@ func TestLogoutFlows(t *testing.T) {
 
 		wg.Wait()
 	})
+
+	t.Run("case=should execute backchannel logout in headless flow with sid", func(t *testing.T) {
+		sid := make(chan string)
+		acceptLoginAsAndWatchSid(t, subject, sid)
+
+		backChannelWG := newWg(1)
+		c := createClientWithBackchannelLogout(t, backChannelWG, func(t *testing.T, logoutToken gjson.Result) {
+			assert.EqualValues(t, <-sid, logoutToken.Get("sid").String(), logoutToken.Raw)
```

Comment:
Here is where my golang channels knowledge is clearly not sufficient.

This test hangs because channel var `sid` is being read twice(L517 and L522) while being written once.

Not sure how to make it work with using the value 2 times here (once for calling the headless logout and once for verifying the `sid` claim in the logout token)

Reply:
What is it that you're trying to achieve?

Reply:
I need `sid` value in 2 places - to pass in the headless logout request as a query param, and in the L517 where we verify the value inside of the `logout_token`.